### PR TITLE
Update the introduction to UCASE() and UNHEX()

### PR DESCRIPTION
**Pull request by lilin90**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update the introduction of UCASE() and UNHEX(), fix https://github.com/pingcap/docs-cn/issues/15952

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16168
- Other reference link(s): https://docs.pingcap.com/zh/tidb/dev/string-functions

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: functions-and-operators/string-functions.md [hunk 1]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
...
+>
+> When the string is `NULL`, the `UCASE()` function returns `NULL`.
+
+Example:
+
+```sql
+SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;
+```
+
+Result:
```
Comment (lilin90) [inline functions-and-operators/string-functions.md:1403]:
```suggestion
Output:
```

File: functions-and-operators/string-functions.md [hunk 2]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
...
+>
+> The argument string must be a valid hexadecimal value that contains `0`~`9`, `A`~`F`, `a`~`f`. If the argument string is `NULL` or falls outside this range, it returns `NULL`.
+
+Example:
+
+```sql
+SELECT UNHEX('54694442');
+```
+
+Result:
```
Comment (lilin90) [inline functions-and-operators/string-functions.md:1427]:
```suggestion
Output:
```

File: functions-and-operators/string-functions.md [hunk 3]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
...
+```
 
 ### [`UNHEX()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_unhex)
 
-Return a string containing hex representation of a number.
+The `UNHEX()` function performs the reverse operation of the `HEX()` function. It treats each pair of characters in the argument as a hexadecimal number and converts it to the character represented by that number, returning the result as a binary string.
+
+> **Note:**
+>
+> The argument string must be a valid hexadecimal value that contains `0`~`9`, `A`~`F`, `a`~`f`. If the argument string is `NULL` or falls outside this range, it returns `NULL`.
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:1419]:
```suggestion
> The argument must be a valid hexadecimal value that contains `0`~`9`, `A`~`F`, or `a`~`f`. If the argument is `NULL` or falls outside this range, the function returns `NULL`.
```

File: functions-and-operators/string-functions.md [hunk 4]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
...
+
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:1420]: The GitHub preview looks weird. Let me set up a doc website preview.

Comment (qiancai) [reply functions-and-operators/string-functions.md:1420]: The website preview looks OK. 
https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/tidb/dev/string-functions#unhex
image

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: functions-and-operators/string-functions.md (hunk 4 above)
Comment (lilin90) [reply functions-and-operators/string-functions.md:1420]: @qiancai Thanks! I just tried another format using "-", which shows well on the GitHub page.

File: functions-and-operators/string-functions.md [hunk 5]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
...
+```
 
 ### [`UNHEX()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_unhex)
 
-Return a string containing hex representation of a number.
+The `UNHEX()` function performs the reverse operation of the `HEX()` function. It treats each pair of characters in the argument as a hexadecimal number and converts it to the character represented by that number, returning the result as a binary string.
+
+> **Note:**
+>
+> The argument must be a valid hexadecimal value that contains `0`-`9`, `A`-`F`, or `a`-`f`. If the argument is `NULL` or falls outside this range, the function returns `NULL`.
```
Comment (lilin90) [inline functions-and-operators/string-functions.md:1419]:
```suggestion
> The argument must be a valid hexadecimal value that contains `0`–`9`, `A`–`F`, or `a`–`f`. If the argument is `NULL` or falls outside this range, the function returns `NULL`.
```

**Review by qiancai — APPROVED** [review]

File: functions-and-operators/string-functions.md [hunk 6]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
...
+The `UCASE()` function is used to convert a string to uppercase letters. This function is equivalent to the `UPPER()` function.
+
+> **Note:**
+>
+> When the string is `NULL`, the `UCASE()` function returns `NULL`.
+
+Example:
+
+```sql
+SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;
```
Comment (xzhangxian1008) [inline functions-and-operators/string-functions.md:1400]: It seems that there is no `UCASE` in example.

Comment (lilin90) [inline functions-and-operators/string-functions.md:1400]:
```suggestion
SELECT UCASE('bigdata') AS result_upper, UCASE('null') AS result_null;
```

File: functions-and-operators/string-functions.md [hunk 7]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
...
+The `UCASE()` function is used to convert a string to uppercase letters. This function is equivalent to the `UPPER()` function.
+
+> **Note:**
+>
+> When the string is `NULL`, the `UCASE()` function returns `NULL`.
+
+Example:
+
+```sql
+SELECT UCASE('bigdata') AS result_upper, UCASE('null') AS result_null;
```
Comment (lilin90) [inline functions-and-operators/string-functions.md:1400]: To keep it consistent with the example for `UPPER()` https://github.com/pingcap/docs/pull/16534/files#r1502244394
```suggestion
SELECT UCASE('bigdata') AS result_upper, UCASE(null) AS result_null;
```

File: functions-and-operators/string-functions.md [hunk 8]
```diff
@@ -1388,11 +1388,51 @@ Remove leading and trailing spaces.
 
 ### [`UCASE()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_ucase)
 
-Synonym for `UPPER()`.
+The `UCASE()` function is used to convert a string to uppercase letters. This function is equivalent to the `UPPER()` function.
+
+> **Note:**
+>
+> When the string is `NULL`, the `UCASE()` function returns `NULL`.
```
Comment (lilin90) [inline functions-and-operators/string-functions.md:1395]:
```suggestion
> When the string is null, the `UCASE()` function returns `NULL`.
```

**Review by xzhangxian1008 — APPROVED** [review]
LGTM

**Pull request merged**